tensorflow_probability/python/distributions/empirical.py

```diff
@@ -25,7 +25,7 @@
 from tensorflow_probability.python.internal import samplers
 from tensorflow_probability.python.internal import tensor_util
 from tensorflow_probability.python.internal import tensorshape_util
-
+from tensorflow_probability.python.stats import percentile
```

Comment:
Our style guide forbids us from importing functions directly, so this should just import stats and then later on the call should be `stats.percentile`